db/install.xml

```diff
@@ -20,6 +20,7 @@
         <FIELD NAME="emailsubjectmanager" TYPE="char" LENGTH="255" NOTNULL="true" SEQUENCE="false" COMMENT="Email subject line for emails going to user's manager"/>
         <FIELD NAME="emailrecipient" TYPE="int" LENGTH="4" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="Who should receive any email - user|manager|both"/>
         <FIELD NAME="duration" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="604800" SEQUENCE="false" COMMENT="how long the reengagement is"/>
+        <FIELD NAME="emailfrequency" TYPE="int" LENGTH="3" NOTNULL="true" DEFAULT="1" SEQUENCE="false" COMMENT="how many times to send the email"/>
```

Comment:
"frequency" to me means 'occurrences per second', (or per minute, hour etc).

I'd really like this to be called 'remindercount' or 'nagmagnitude' or something